functions/firebase/src/main/java/com/example/functions/firebase/FirebaseFirestoreReactive.java

```diff
@@ -33,15 +33,15 @@
 public class FirebaseFirestoreReactive implements RawBackgroundFunction {
 
   // Use GSON (https://github.com/google/gson) to parse JSON content.
-  private Gson gsonParser = new Gson();
+  private Gson GSON_PARSER = new Gson();
```

Comment:
This isn't static, or final. (nit: Also I almost always see it called just `gson`)

Reply:
Will fix this more broadly.